src/openzaak/components/catalogi/tests/test_zaaktype.py

```diff
@@ -1421,6 +1430,136 @@ def test_update_published_zaaktype_requires_forced_write_scope(self):
+        }
+        self.assertEqual(result["broncatalogus"], expected_broncatalogus)
+
+        expected_bronzaaktype = {
+            "url": f"http://testserver{get_operation_url('zaaktype_read', uuid=bron_zaaktype.uuid)}",
+            "identificatie": bron_zaaktype.identificatie,
+            "omschrijving": bron_zaaktype.zaaktype_omschrijving,
+        }
+        self.assertEqual(result["bronzaaktype"], expected_bronzaaktype)
+
+    def test_put_update_zaaktype_with_bron_fields(self):
+        catalog1 = CatalogusFactory.create(domein="ABC", rsin="517439943")
+        bron_zaaktype = ZaakTypeFactory.create(catalogus=catalog1)
+
+        catalog2 = CatalogusFactory.create(domein="XYZ", rsin="004455667")
+        zaaktype = ZaakTypeFactory.create(catalogus=catalog2)
+        zaaktype_url = reverse(zaaktype)
+
+        data = {
+            "identificatie": 0,
+            "doel": "update bron fields test",
+            "aanleiding": "PUT bron test",
+            "indicatieInternOfExtern": InternExtern.extern,
+            "handelingInitiator": "indienen",
+            "onderwerp": "Klacht",
+            "handelingBehandelaar": "uitvoeren",
+            "doorlooptijd": "P30D",
+            "opschortingEnAanhoudingMogelijk": False,
+            "verlengingMogelijk": True,
+            "verlengingstermijn": "P30D",
+            "publicatieIndicatie": True,
+            "verantwoordingsrelatie": [],
+            "productenOfDiensten": ["https://example.com/product/123"],
+            "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduiding.openbaar,
+            "omschrijving": "PUT bron test",
+            "gerelateerdeZaaktypen": [],
+            "referentieproces": {"naam": "ReferentieProces 0", "link": ""},
+            "catalogus": reverse(catalog2),
+            "besluittypen": [],
+            "beginGeldigheid": "2018-01-01",
+            "versiedatum": "2018-01-01",
+            "verantwoordelijke": "063308836",
+            "broncatalogus": {
+                "url": f"http://example.com{reverse(catalog1)}",
+                "domein": catalog1.domein,
+                "rsin": catalog1.rsin,
+            },
+            "bronzaaktype": {
+                "url": f"http://example.com{reverse(bron_zaaktype)}",
+                "identificatie": bron_zaaktype.identificatie,
+                "omschrijving": bron_zaaktype.zaaktype_omschrijving,
+            },
+        }
+
+        response = self.client.put(zaaktype_url, data, format="json")
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+
+        zaaktype.refresh_from_db()
+        self.assertEqual(zaaktype.broncatalogus_domein, catalog1.domein)
+        self.assertEqual(zaaktype.broncatalogus_rsin, catalog1.rsin)
+        self.assertEqual(
+            zaaktype.bronzaaktype_identificatie, bron_zaaktype.identificatie
+        )
+        self.assertEqual(
+            zaaktype.bronzaaktype_omschrijving, bron_zaaktype.zaaktype_omschrijving
+        )
```

Comment:
Could you also add the assertions for `broncatalogus_url` and `bronzaaktype_url` here?